### PR TITLE
Restore all end-to-end replication tests

### PR DESCRIPTION
**Pull request by acco**

### CODE REVIEW

**Comment by acco** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/113?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#113** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/113?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#112** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/112?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#111** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/111?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#110** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/110?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @acco and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: test/sequin/postgres_replication_test.exs [hunk 1]
```diff
@@ -112,230 +121,183 @@ defmodule Sequin.PostgresReplicationTest do
...
+      assert_receive {ReplicationExt, :flush_messages}, 500
+
+      # Update the character
+      UnboxedRepo.update!(Ecto.Changeset.change(character, planet: "Arrakis"))
+
+      # Wait for the update message to be handled
+      assert_receive {ReplicationExt, :flush_messages}, 500
+
+      # Fetch consumer events
+      [_insert_event, update_event] = Consumers.list_consumer_events_for_consumer(consumer.id)
```
Comment (RTLS) [inline test/sequin/postgres_replication_test.exs:138]: should we order this query? by inserted at asc?

File: test/support/models/character_2pk.ex [hunk 2]
```diff
@@ -13,6 +13,15 @@ defmodule Sequin.Test.Support.Models.Character2PK do
...
   end
 
+  def table_oid do
+    Sequin.Repo.one(
+      from(pg in "pg_class",
+        where: pg.relname == "characters_2pk",
+        select: pg.oid
+      )
+    )
+  end
```
Comment (RTLS) [inline test/support/models/character_2pk.ex:23]: could probably have a function that takes any ecto model and returns the oid

```
iex(sequin-stream-dev@carters-mbp)8> c = %Sequin.Consumers.HttpPushConsumer{}
%Sequin.Consumers.HttpPushConsumer{
  __meta__: #Ecto.Schema.Metadata<:built, "sequin_config", "http_push_consumers">,
  id: nil,
  name: nil,
  backfill_completed_at: nil,
  ack_wait_ms: 30000,
  max_ack_pending: 10000,
  max_deliver: nil,
  max_waiting: 20,
  message_kind: :record,
  status: :active,
  source_tables: [],
  account_id: nil,
  account: #Ecto.Association.NotLoaded<association :account is not loaded>,
  http_endpoint_id: nil,
  http_endpoint: #Ecto.Association.NotLoaded<association :http_endpoint is not loaded>,
  replication_slot_id: nil,
  replication_slot: #Ecto.Association.NotLoaded<association :replication_slot is not loaded>,
  inserted_at: nil,
  updated_at: nil
}
iex(sequin-stream-dev@carters-mbp)9> c.__meta__.prefix
"sequin_config"
iex(sequin-stream-dev@carters-mbp)10> c.__meta__.source
"http_push_consumers"
```

**Review by RTLS — APPROVED** [review]

File: test/sequin/postgres_replication_test.exs (hunk 1 above)
Comment (acco) [reply test/sequin/postgres_replication_test.exs:138]: Good call

File: test/support/models/character_2pk.ex (hunk 2 above)
Comment (acco) [reply test/support/models/character_2pk.ex:23]: I like it, will do in follow-up

**Pull request merged**